kong/init.lua

```diff
@@ -1831,6 +1831,10 @@ local function serve_content(module)
 
   ngx.header["Access-Control-Allow-Origin"] = ngx.req.get_headers()["Origin"] or "*"
 
+  if kong.configuration.log_level == "debug" then
+    ngx.header['X-Kong-Test-Transaction-ID'] = kong_global.get_current_transaction_id()
```

Comment:
style, just double quotes:
```suggestion
    ngx.header["X-Kong-Test-Transaction-ID"] = kong_global.get_current_transaction_id()
```